priv/templates/extended_bin

```diff
@@ -642,28 +623,26 @@ case "$1" in
         export EMU
         export PROGNAME
 
-        # Store passed arguments since they will be erased by `set`
-        ARGS="$@"
```

Comment:
FYI, there is no way to preserve double-quotes and space characters *and* re-assign the positional parameter variable. Believe me, I tried. The side-effect of this is that the `erlexec` command must be included twice below - once for logging, and once "for real".